From 6b48c79467f7dae7e00f9e21211e97acda0d2454 Mon Sep 17 00:00:00 2001
From: Zhong Hongbo <hongbo.zhong@windriver.com>
Date: Thu, 29 May 2014 11:32:53 +0800
Subject: [PATCH 71/71] axxm55xx: arm: Fix package fail for lsi

ERROR: QA Issue: non debug package contains .debug directory: kernel-dev path kernel/arch/arm/boot/.debug/linux.img
ERROR: QA run found fatal errors. Please consider fixing them.

Since the commit(3153818ccf) disable the line to
remove the middle image, So revert the change

The commit information as the following:
commit 3153818ccfa508ebd833b7fc65ff86088ccc74b6
Author: Anders Berg <anders.berg@lsi.com>
Date:   Mon Jan 20 13:50:20 2014 +0100

    mach-axxia: Split device tree into SoC/board parts

    https://github.com/lsigithub/lsi_axxia_yocto_public_3.10.git standard/axxia/base
    commit 8a9c1c875584d8c4ab9df652504e0e94cb16fc13

    Split up the device tree into several files. The axm55xx.dtsi contains all the
    SoC device nodes, the axm55<NN>-cpus.dtsi contains the CPU nodes with <NN>
    number of CPU cores. The top device tree for the AXM5516 validation board (aka.
    Amarillo) then includes the axm55xx.dtsi, the axm5516-cpus.dtsi and completes
    the device nodes with board specific configuration.

    Signed-off-by: Anders Berg <anders.berg@lsi.com>

Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/boot/fmboot/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/fmboot/Makefile b/arch/arm/boot/fmboot/Makefile
index 5b046c0..6b71905 100644
--- a/arch/arm/boot/fmboot/Makefile
+++ b/arch/arm/boot/fmboot/Makefile
@@ -13,7 +13,7 @@ clean:
 arch/arm/boot/linux.img: $(obj)/fmboot.o $(srctree)/$(obj)/fmboot.lds $(obj)/zImage.fm
 	cd $(obj) && $(LD) -o ../linux.img --script=$(srctree)/$(obj)/fmboot.lds
 	tar jcf $(obj)/../linux.img.tar.bz2 $(obj)/../linux.img
-	#rm -rf $(obj)/../linux.img
+	rm -rf $(obj)/../linux.img
 
 $(obj)/zImage.fm: $(ZIMAGE) $(obj)/axm5504-sim.dtb
 	python $(srctree)/$(obj)/pack.py $(ZIMAGE) $(obj)/axm5504-sim.dtb > $@
-- 
1.7.5.4

